packages/ui/src/kit/VCircularLoader/VCircularLoader.vue

```diff
@@ -0,0 +1,9 @@
+<script>
```

Comment:
and maybe not circular-loader, because for loading we use progress-circle
maybe name to something else?

Reply:
This name is enough semantic and means just circular loader, which can display progress for you. IMO there is not any reason for new name search